src/mock/main.py

```diff
@@ -0,0 +1,29 @@
+"""This module is for thread start."""
+
+from pybitmessage import state
+from pybitmessage.bitmessagekivy.mpybit import NavigateApp
+from class_addressGenerator import FakeAddressGenerator
+# from class_sqlThread import sqlThread
+
+
+def main():
+    """main method for starting threads"""
+    if state.enableObjProc:
+        # Start the address generation thread
+        addressGeneratorThread = FakeAddressGenerator()
+        # close the main program even if there are threads left
+        addressGeneratorThread.daemon = True
+        addressGeneratorThread.start()
+
+        # sqlLookup = sqlThread()
```

Comment:
this can be removed, once we have a mock sql thread we can re-add.

Reply:
ok